docs/RELEASE.md

```diff
@@ -1,39 +1,29 @@
 # Release process
 
-* Ensure you have a backup of all working files and then remove files not tracked by git
-  `git clean -xdf`. **NOTE**: this will delete all files in the tuf tree that aren't
-  tracked by git
```

Comment:
removing this seems right, but there is a related issue: https://github.com/theupdateframework/python-tuf/issues/1942 (build results for verify_release are still affected by randome files in tree).

I can have a look at the workaround (making a clean checkout before running build).

Reply:
made #1947 for this

Reply:
Thanks, I considered leaving that line, but secretly hoped that we could find a different fix (`git clean -xdf` already killed my pre commit hook once). https://github.com/theupdateframework/python-tuf/pull/1947 looks great for this.

Reply:
1947 is merged, it's all good now I think :crossed_fingers: